src/renderer/src/api.js

```diff
@@ -53,6 +55,7 @@ export const callApiStartLogs = async (id) => {
   return await window.api.startLogs(id)
 }
 export const getAppLogs = callback => window.api.onAppLog(callback)
+export const removeAppLogs = callback => window.api.removeAppLog(callback)
```

Comment:
I don't think you need to pass the listener

Reply:
Done!